.gitignore

```diff
@@ -3,3 +3,4 @@
 behat.yml
 vendor
 composer.lock
+.idea
```

Comment:
you should add this to your global gitignore or the not commited git ignore (`.git/info/exclude`). It should not be part of the project's gitignore.